main.cpp

```diff
@@ -50,10 +50,18 @@
 #include "daemon/DaemonManager.h"
 #include "AddressBook.h"
 #include "model/AddressBookModel.h"
+#include "wallet/wallet2_api.h"
 
+void messageHandler(QtMsgType type, const QMessageLogContext &context, const QString &msg)
+{
+    // Send all message types to logger
+    Monero::Wallet::debug(msg.toStdString());
+}
```

Comment:
OK for now with same category as wallet API (WalletAPI). Could update that later on monero side.